src/main/java/cz/vutbr/web/csskit/OutputUtil.java

```diff
@@ -69,6 +69,31 @@ public static StringBuilder appendTimes(StringBuilder sb, String append, int tim
 	}
 	
 	/**
+	 * Appends all elements of <code>array</code> to buffer, separated by delimiter
+	 * @param <T> Type of elements stored in <code>array</code>
+	 * @param sb StringBuilder to be modified
+	 * @param array Array of elements
+	 * @param delimiter Delimiter to separate elements
+	 * @return Modified <code>sb</code> to allow chaining
+	 */
+	public static <T> StringBuilder appendArray(StringBuilder sb, T[] array, String delimiter) {
+		
+		boolean firstRun = true;
```

Comment:
Oh, I am not sure. I just copied the `appendList()` function and changed the parameter type to array.

If you think this still needs change, please let me know. I will analyse it further.